.github/workflows/wasmcloud.yml

```diff
@@ -434,6 +434,7 @@ jobs:
   nfpm:
     if: startsWith(github.ref, 'refs/tags/wash-cli-v')
     env:
+      REF: ${{ github.ref }}
```

Comment:
Odd that this is okay with `refs/tags`?

Reply:
I don't follow, do you mean that it's okay that this has the refs/tags prefix? We take care of that a little below
```
export VERSION=$(echo $REF | cut -d/ -f3)
```

Reply:
Ah missed that, thanks!